lib/new_relic/agent/log_event_aggregator.rb

```diff
@@ -160,6 +180,64 @@ def enabled?
 
       private
 
+      def add_custom_attribute(key, value)
```

Comment:
I may make a separate PR when this wraps up to organize all the private methods alphabetically.